source/blog/2017-11-01-async-stack-traces.md

```diff
@@ -31,8 +31,10 @@ only Scala 2.11.  We are not working on capture points for 2.12 at this time;
 if you would like to help make this feature available for more users, you can
 submit a pull request on [util's GitHub repo](https://github.com/twitter/util).
 
-Lastly, note that the capture points XML file will be published in our next
-release.
+Lastly, note that the capture points XML file will be published as an artifact 
```

Comment:
```
"To start using, you can grab the capture points XML file [directly from Github](link to file here), and after our next release it will be available as a artifact published to Maven Central" 
```

something along those line? blah, words.